tests/baselines/reference/privateNameFieldCallExpression.js

```diff
@@ -0,0 +1,29 @@
+//// [privateNameFieldCallExpression.ts]
+class A {
+    #fieldFunc = () => this.x = 10;
```

Comment:
this test would be more believable with `function () { this.x = 10 }` since that's where correct emit would make a difference